src/helpers.ts

```diff
@@ -62,22 +63,19 @@ function hasNodeExecutable(rootPath: string, executable: string): boolean {
  */
 export function pathToJest(pluginSettings: IPluginSettings) {
   if (pluginSettings.pathToJest) {
-    if (isBootstrappedWithCRA(pluginSettings.rootPath)) {
-      return 'npm test --'
-    }
     return normalize(pluginSettings.pathToJest)
   }
 
-  const platform = process.platform
-  if (platform === 'win32' && existsSync(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest.cmd'))) {
-    return normalize(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest.cmd'))
-  } else if (
-    (platform === 'linux' || platform === 'darwin') &&
-    existsSync(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest'))
-  ) {
-    return normalize(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest'))
+  if (isBootstrappedWithCRA(pluginSettings.rootPath)) {
+    return 'npm test --'
+  }
+
+  const defaultPath = normalize('node_modules/.bin/jest')
+  if (existsSync(join(pluginSettings.rootPath, defaultPath))) {
+    return defaultPath
   }
 
+  vscode.window.showWarningMessage("Jest couldn't be found. Maybe you forgot to run `npm install`?")
   return 'jest'
```

Comment:
Neat idea. Should we worry about how noisy this could end up being for users who have installed Jest globally or are working with a package in a monorepo where `node_modules` isn't in the workspace root (like Jest)? I think we can build on this and lead users to helpful documentation about how to configure the extension. 👍

Reply:
Probably, especially because users using a global Jest install at the moment also see the "This extension relies on Jest 20+ features…" error message.
It was just that opening VS Code before `npm install` finished happened to me more often than thinking about global Jest installs.